composer.json

```
@@ -36,6 +35,7 @@
         "symplify/composer-json-manipulator": "dev-master",
         "symplify/console-color-diff": "dev-master",
         "symplify/package-builder": "dev-master",
+        "symplify/php-config-printer": "^0.4.2",
```

`dev-master` should be used instead, as other `symplify/*` packages. It's only temporary, till Symplify 9 gets release in a week or two

```suggestion
        "symplify/php-config-printer": "dev-master",
```